nunaliit2-couch-onUpload/src/main/java/ca/carleton/gcrc/couch/onUpload/inReach/InReachProcessorImpl.java

```diff
@@ -49,6 +56,37 @@ public void performSubmission(FileConversionContext conversionContext) throws Ex
 					}
 				}
 			}
+
+			int emergencyState = jsonItem.optInt("EmergencyState", -12345);
```

Comment:
Is there a reason not to use has method in all these places instead of opt like this:
```
if(jsonItem.has("EmergencyState")) {
  genericInReachSchema.put("EmergencyState", jsonItem.get("EmergencyState"))
}
```